init/src/initDatabase.ts

```diff
@@ -0,0 +1,30 @@
+import { readFileSync } from 'fs';
+import { Client } from 'pg';
+import { environment as env } from '../../serverless.environment';
+
+export const initDatabase = async () => {
+  const file = __filename.replace('.ts', '.sql');
```

Comment:
[Minor]
Not obvious. I suggest to explicitly require a file you need 🙂 .

Reply:
agreed